src/context/mod.rs

```diff
@@ -476,9 +490,16 @@ impl<'a, T: This> CallContext<'a, T> {
     }
 
     /// Produces a handle to the `this`-binding.
+    #[cfg(feature = "legacy-runtime")]
     pub fn this(&mut self) -> Handle<'a, T> {
         Handle::new_internal(T::as_this(self.info.this(self)))
     }
+
+    /// Produces a handle to the `this`-binding.
+    #[cfg(feature = "napi-runtime")]
+    pub fn this(&mut self, env: Env) -> Handle<'a, T> {
```

Comment:
Does this need an `env` parameter or can it be obtained with `self.env()`?

Reply:
Ohh good call. I used the `ctx.env()` method elsewhere but forgot about it here.